.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+.yarn-offline-mirror
```

Comment:
.yarn-offline-mirror was not added to .gitignore on purpose. It should be included to the committed files as a best practice, so that it can speed up CI builds and so CI builds don't depend on npm being up.

Reply:
ah okay. I thought that binaries in the repo are not that optimal (because it bloat the commit), but to speed up CI builds is a good thing to be aware of :). Thanks for the advice, maybe put them in a separate commit, would be an option, to make the commits more readable.